src/code42cli/cmds/search/extraction.py

```diff
@@ -1,6 +1,5 @@
 import json
```

Comment:
Kiran, this was totally my fault, but can you fix line 26?  The error message is backwards.  It should say `--include-all only allowed for Table output format`.